crates/cja/src/cron/worker.rs

```diff
@@ -21,16 +22,20 @@ impl<AppState: AS> Worker<AppState> {
 
         tracing::debug!("Starting cron loop");
         loop {
-            let last_runs = sqlx::query!(
-                "SELECT name, max(last_run_at) as last_run_at FROM Crons GROUP BY name"
+            let last_runs = sqlx::query(
```

Comment:
I want to use the compile time checks here. We only wanted to not do them in test code paths